src/Compilers/Core/Portable/Syntax/LineSpanMapping.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.Serialization;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Represents a line mapping defined by a single line mapping directive (<c>#line</c> in C# or <c>#ExternalSource</c> in VB).
+    /// </summary>
+    [DataContract]
+    public readonly struct LineMapping : IEquatable<LineMapping>
```

Comment:
Please rename the file to match the name of the struct.

Reply:
Done.